sass/directives/02_base_components/button/_button.scss

```diff
@@ -222,6 +222,101 @@ $massive-button-horizontal-padding: $base-input-font-size * 2;
   }
 }
 
+// Fancy Play Button
+// Initial state
+$ripple-diameter: 24em;
+$play-button-size: 4em;
+$ripple-color: white;
+$ripple-hover-color: #447777;
+$ripple-hover-transition: 1500ms ease;
```

Comment:
Looking over the stylesheet, I agree that the variables mentioned should go into that sheet. I went ahead and moved those, but am using the namespace `video-play-button-ripple` to better identify those variables.

Regarding your second comment, because there is so little in the animation sheet as of now, I think keeping the values as they are would make more sense. Considering that these values, currently, are very specific to this feature.